client/verta/tests/conftest.py

```diff
@@ -267,14 +269,9 @@ def experiment_run(client):
 def repository(client):
     name = _utils.generate_default_name()
     repo = client.get_or_create_repository(name)
-    root_id = repo.get_commit()
 
     yield repo
 
-    try:
-        utils.delete_commit(repository.id, root_id, repository._conn)
-    except:
-        pass  # may have already been deleted in test
```

Comment:
The back end also returns an error when deleting the root, so we're not doing that anymore.

(fun fact! because of the blanket `except:` here, there's actually a `NameError` that's gone uncaught ever since this fixture was written!)

Reply:
Never catch all =)